app/src/main/java/com/framgia/music_40/screen/listmusic/ListMusicFragment.java

```diff
@@ -0,0 +1,52 @@
+package com.framgia.music_40.screen.listmusic;
+
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+import com.framgia.music_40.R;
+import com.framgia.music_40.data.model.Music;
+import com.framgia.music_40.data.source.MusicRepository;
+import com.framgia.music_40.data.source.remote.MusicRemoteDataSource;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListMusicFragment extends Fragment implements OnitemListMusicClick {
+
+    private static final String BUNDLE_LIST = "list";
```

Comment:
change to     private static final String ARGUMENT_LIST_MUSIC = "ARGUMENT_LIST_MUSIC";
convention 2.10 -2.11
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md